spotbugs-annotations/src/main/java/edu/umd/cs/findbugs/annotations/OverridingMethodsMustInvokeSuper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * FindBugs - Find Bugs in Java programs
```

Comment:
For new files, this should be SpotBugs, not FindBugs, and it doesn't belong to the University of Maryland.

@spotbugs/everyone I don't think we've actually discussed what copyright notice to use going forward...